hack/lib/serverless.bash

```diff
@@ -155,6 +155,10 @@ function deploy_knativeserving_cr {
     deploy_with_kourier
   fi
 
+  if [[ $ENABLE_TRACING == "true" ]]; then
```

Comment:
Do we actually need to disable this since by default we enable it? What cases you had in mind?

Reply:
ok